action.yml

```diff
@@ -121,6 +125,21 @@ runs:
             TWINE_OPTS="$TWINE_OPTS --skip-existing"
           fi
           python -m pip install twine
+          if [[ -z "$TWINE_PASSWORD" ]]; then
+            # vis https://docs.pypi.org/trusted-publishers/using-a-publisher/#the-manual-way
+            export TWINE_PASSWORD=$(
+        python -c 'import json, urllib.request as req
+        audience = req.urlopen("'${INPUT_REGISTRY_DOMAIN}'/_/oidc/audience").read()
```

Comment:
Native `getenv` rather than shell variables, please?

Reply:
Why bother with the additional code?

Reply:
Exceedingly defensive; primed by https://github.com/trufflesecurity/trufflehog/pull/2259

Reply:
```sh
$ foo="let's try"
$ echo 'some code "using '${foo}' seems to work" right?'
some code "using let's try seems to work" right?
```

Reply:
oh I see

```
$ bar='let"s try'
$ python -c 'print("using '${bar}' seems to work")'
SyntaxError
```